src/features/customerView/utils.ts

```diff
@@ -170,9 +170,9 @@ export const getBills = (profile: CUSTOMER_PROFILE): (BerthBill | WinterStorageB
           {
             status: orderNode.status,
             berthInformation: {
-              number: lease.berth.number,
-              pierIdentifier: lease.berth.pier.properties?.identifier ?? '',
-              harborName: lease.berth.pier.properties?.harbor?.properties?.name ?? '',
+              number: 0,
+              pierIdentifier: '',
+              harborName: '',
```

Comment:
Could you add a `TODO` comment here to replace the hardcoded values once they are implemented on the backend?

Reply:
Done.